Tella/Scenes/CommonConnectionReport/Views/Draft/DraftView.swift

```diff
@@ -0,0 +1,286 @@
+//
+//  DraftView.swift
+//  Tella
+//
+//  Created by gus valbuena on 6/26/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct DraftView: View  {
+    @StateObject var viewModel: DraftMainViewModel
+    
+    @State private var menuFrame : CGRect = CGRectZero
+    @State private var shouldShowMenu : Bool = false
+    
+    @EnvironmentObject var sheetManager: SheetManager
+      
+    var reportsViewModel : ReportsMainViewModel
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    
+    var body: some View {
+        ContainerView {
+            contentView
+                .environmentObject(viewModel)
```

Comment:
if we are passing the viewmodel in params we an remove the  `.environmentObject(viewModel) `This may lead to crashes